applications/launchpad/gui-react/src/containers/WalletContainer/Send/styles.ts

```diff
@@ -0,0 +1,71 @@
+import styled from 'styled-components'
+
+export const StyledSendForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  padding: ${({ theme }) => theme.spacing(1.67)};
```

Comment:
where do you get these values ^^

Reply:
It's a padding inside of the modal, equal to 40px. I picked this `1.67` to be close to the designs.